docs: fix and improve multiple sections (#4396)
nuxt/framework: docs/content/3.api/1.composables/use-request-headers.md

```

::alert{icon=👉}

> On the client side, `useRequestHeaders` will return an empty object.

Reviewer: ```suggestion
In the browser, `useRequestHeaders` will return an empty object.
```
